src/components/ProductCategory/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Heading, UnorderedList } from '@chakra-ui/react';
+import { ProductItem } from '@/components/ProductItem';
+
+type CategoryWithItems = {
+    title: string,
+    items: string[],
+}
+
+export const ProductCategory = (productCategory: CategoryWithItems) => {
+    const { title, items } = productCategory;
+    return (<Box m='47px 89px 47px 80px'>
+        <Heading
+            as='h3'
+            noOfLines={1}
+            fontSize='18px'
+            fontWeight='500'
+            mb='18px'
+        > {title} </Heading>
```

Comment:
The right indentation is like below example

```suggestion
        <Heading
            as='h3'
            noOfLines={1}
            fontSize='18px'
            fontWeight='500'
            mb='18px'
        > 
          {title} 
        </Heading>
```

Reply:
Your editor is using 4 spaces to indentation, is important change that for 2 spaces. (Its a my mistake, because I forget to create .editorconfig file to define this default for the project)

Check if prettier and eslint working right on your code editor.